src/components/Page/Base.js

```diff
@@ -25,12 +28,14 @@ BasePage.propTypes = {
     PropTypes.arrayOf(PropTypes.node),
     PropTypes.node,
   ]),
-  footerProps: PropTypes.shape({}),
+  menus: PropTypes.shape({
+    footerMenu: PropTypes.arrayOf({}),
```

Comment:
```suggestion
    footerMenu: PropTypes.arrayOf(PropTypes.shape({})),
```
Warning: Failed prop type: Property `menus.footerMenu` of component `BasePage` has invalid PropType notation inside arrayOf.